src/libmongoc/doc/mongoc_client_encryption_get_key.rst

```diff
@@ -0,0 +1,32 @@
+:man_page: mongoc_client_encryption_get_key
+
+mongoc_client_encryption_get_key()
+==================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+   mongoc_cursor_t *
```

Comment:
Relates to https://github.com/mongodb/specifications/pull/1229#discussion_r880803965